modules/directives/src/main/scala/scala/build/errors/WrongJavaHomePathError.scala

```diff
@@ -0,0 +1,7 @@
+package scala.build.errors
+
+class WrongJavaHomePathError(exceptionMessage: String)
+    extends BuildException(message =
+      s"""The java home path argument in the using directives at could not be found!
+         |$exceptionMessage""".stripMargin) {}
+//TODO "at where?"
```

Comment:
TODO?

Reply:
yes… it has to include the address. There should be a dedicated argument for that.